### PR TITLE
Proposed change in app description

### PR DESCRIPTION
**Pull request by Ilithy**
Hi,
* there were twists that I found strange, I allow myself to offer you an alternative version of the general presentation of the app (I only touched on that).

* I tried to limit repetitions (especially words: links, urls, app)

_I hope you like it, if not don't hesitate for a second to reject this_

### CODE REVIEW

**Comment by TrianguloY** [comment]
I admit I wasn't very inspired, thanks for the suggestion!
I made changes-over-your-changes too (mainly punctuation). Check them in case you want to tweak further and I'll merge it!

**Comment by TrianguloY** [comment]
Btw, would you like to have writing permission in the repository (aka be a collaborator)?
You will be able to make changes directly, without any PR, so feel free to fix typos, update screenshots, tweak strings, etc (even coding if you try! it's fun!). For relatively big changes you can still make PRs as usual (creating branches here or in your fork) but in any case this is git so if I don't like something I can revert it by pressing a button, don't feel pressured thinking I won't like it (I will, but I can change it if not).

Note: I can add you as collaborator any time, in case you think you are not prepared yet, even though you are. Just tell me when you want and I'll add you.

File: app/src/main/play/listings/en-US/full-description.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ allowing you to make changes to it as well.
 Especially useful when you need to open an external link either from an email, a social network app or others.
 
 
-Url Checker is developed by TrianguloY originally for personal use, it is open source (under CC BY 4.0 license), free, without any ads or trackers, light sized and using as few permissions as workable (only the internet permission, for module checks that will only be performed when the user request them). The source code is available on GitHub too if you want to suggest changes, perform modifications or propose translation: https://github.com/TrianguloY/UrlChecker
+Url Checker is developed by TrianguloY, originally for personal use. It is open source (under CC BY 4.0 license), free, without any ads or trackers, light sized and using as few permissions as necessary (only the internet permission, for module checks that will only be performed when the user request them). The source code is available on GitHub too, if you want to suggest changes, perform modifications or propose translations: https://github.com/TrianguloY/UrlChecker
```
Comment (Ilithy) [inline app/src/main/play/listings/en-US/full-description.txt:5]: It's not really useful to have the word "translations" in the plural, in my opinion (one person -> one translation, most of the time)

Comment (TrianguloY) [reply app/src/main/play/listings/en-US/full-description.txt:5]: Hmm, most of the time, but on my Click to Chat app I think there wer 1 or 2 users that gave me two translations each.
For singuular you need to add 'a' (I think).
Hmm, what about:

`... perform modifications or provide a new translation: https://github.com/TrianguloY/UrlChecker`

**Comment by Ilithy** [comment]
>I made changes-over-your-changes too (mainly punctuation). Check them in case you want to tweak further and I'll merge it!

I looked, and it's great changes, thank you

> Btw, would you like to have writing permission in the repository (aka be a collaborator)? You will be able to make changes directly, without any PR, so feel free to fix typos, update screenshots, tweak strings, etc (even coding if you try! it's fun!). For relatively big changes you can still make PRs as usual (creating branches here or in your fork) but in any case this is git so if I don't like something I can revert it by pressing a button, don't feel pressured thinking I won't like it (I will, but I can change it if not).
> 
> Note: I can add you as collaborator any time, in case you think you are not prepared yet, even though you are. Just tell me when you want and I'll add you.

Thank you very much, I'm willing, but not immediately (with the different PRs, I don't want to do anything stupid)

File: app/src/main/play/listings/en-US/full-description.txt (hunk 1 above)
Comment (Ilithy) [reply app/src/main/play/listings/en-US/full-description.txt:5]: very good, it's not a critical point

**Comment by TrianguloY** [comment]
Now that the PRs are merged, I sent you the invitation to become a collaborator. You can accept it whenever you want.
Any question don't hesitate to ask!

**Comment by Ilithy** [comment]
Thank you, I have accepted it, and sent you one for my fork (you can do what you like).
For my part, most of the time, I will ask you for a validation (by review) except for small modifications (like screenshots), I don't want to make mistakes, or modifications that you don't agree with (and that will force you to correct them), I will contact you (via discussions, not to pollute your mailbox) if I have any doubts, or questions

**Comment by TrianguloY** [comment]
> Thank you, I have accepted it, and sent you one for my fork (you can do what you like).

I accepted it, for testing, but as you probably saw here on GitHub when you make an PR the owner of the destination repository gains access to that branch (to update the PR).
The idea now is that you can do all new changes directly on this repo, create as many branches as you want. This has the benefits that you can better rebase/update/merge. Of course you can still use your fork if you prefer.

> For my part, most of the time, I will ask you for a validation (by review) except for small modifications (like screenshots), I don't want to make mistakes, or modifications that you don't agree with (and that will force you to correct them), I will contact you (via discussions, not to pollute your mailbox) if I have any doubts, or questions.

Perfect, but as I said I can revert any change just by clicking a button so don't worry about "wrong" changes (If I need to make changes items the same work for me to do it directly on the master branch or the PR)

**Pull request merged**